src/tips/en/show-web-vitals.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2023-01-06
+authors: Kilian Valkhof
+title: Show web vitals
```

Comment:
Web vitals are also available in chromium browsers. I think they might only be available in the Performance tool though (unless if they're also in the Performance Insights panel in Chrome). In any case, I think this tip applies not just to Polypane. It would be good to extend it to other browsers.
What do you think?

Reply:
I don't really know how to explain the performance panel (there's a checkbox, you press the reload button and then it shows fcp and lcp timing)

The performance insights I think you need to enable? It's in my Chrome installation but not in my Chromium installation. That one shows FCP, LCP and CLS with a details panel.

I can describe what I see but I probably wouldn't do a good job. Want me to do a first pass and for someone else to clean it up, or someone else to add it to this tip?

Reply:
Ok, that's fair, let's not add anything for chrome or chromium then, and keep this tip for polypane only. We can update it later.